src/js/components/Grid/__tests__/Grid-test.js

```diff
@@ -309,4 +309,54 @@ describe('Grid', () => {
     const tree = component.toJSON();
     expect(tree).toMatchSnapshot();
   });
+
+  test('width', () => {
+    const component = renderer.create(
```

Comment:
Good tests. 
I think we are trying to avoid the usage of `renderer.create` from `react-test-renderer` library when adding new tests. We should use `{ render } from '@testing-library/react'` instead.